src/app/profiles/profile-detail/profile-detail.component.html

```diff
@@ -31,7 +31,27 @@
               </mat-option>
             </mat-select>
           </mat-form-field>
-          <div fxFlex="50" style="margin-top:15px">
+          <div fxFlex="100">
+            <mat-label>Redirection</mat-label>
+          </div>
+          <mat-form-field fxFlex="100">
+            <mat-label>User Consent</mat-label>
```

Comment:
Could add a `<mat-hint>` that talks about what user consent is

```suggestion
<mat-label>User Consent</mat-label>
<mat-hint>Controls when a user must be present for select AMT features</mat-hint>
```